packages/react/src/components/header/components/headerActionBarItem/HeaderActionBarItem.module.scss

```diff
@@ -30,8 +31,8 @@ button.actionBarItem {
 }
```

Comment:
The focus styles for action bar items are absolutely bananas, on Chome at least.
<img width="88" alt="Screenshot 2023-08-11 at 14 36 20" src="https://github.com/City-of-Helsinki/helsinki-design-system/assets/108321134/dbc31203-7f06-4af1-9bde-2cf46a973b41">
<img width="69" alt="Screenshot 2023-08-11 at 14 36 16" src="https://github.com/City-of-Helsinki/helsinki-design-system/assets/108321134/1fa62465-c1ed-4b08-bf5c-55c1498b45ea">
<img width="69" alt="Screenshot 2023-08-11 at 14 36 09" src="https://github.com/City-of-Helsinki/helsinki-design-system/assets/108321134/76ceee92-a9a1-42ae-9fde-638d082b5a27">

Reply:
damn. fix for visual tests broke that, I didn't remember to check. I'll find a fix for that

Reply:
Fixed.